### PR TITLE
fix(charts): quick fix to avoid undefined throwing

### PR DESCRIPTION
**Pull request by dmgauthier**
### Proposed Changes

Quickly fixed the `cannot apply method of undefined` errors that were throwing in the demo.
I did not took the time to really understand the components or the methods that were used, but I could check deeper if it worth the time.

plus adding some unit tests if need be.

**Edit:** After more research. It was the tooltip rendering that was at fault when not finding the parent component.

I did not change the implementation. Only avoided any undefined in the workflow.

### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/react-vapor/src/components/drop/DropPod.tsx [hunk 1]
```diff
@@ -83,8 +83,8 @@ class RDropPod extends React.PureComponent<IRDropPodProps, IDropPodState> {
             };
         } else {
             this.lastPosition = {
-                position: DropPodPosition.bottom,
-                orientation: DropPodPosition.left,
+                position: DropPodPosition?.bottom,
+                orientation: DropPodPosition?.left,
```
Comment (gdostie) [inline packages/react-vapor/src/components/drop/DropPod.tsx]: You can remove the `?` here, `DropPodPosition` is a `const` with the following keys (which will always be defined).
```ts
export const DropPodPosition = {
    bottom: 'bottom',
    top: 'top',
    left: 'left',
    right: 'right',
};
```

**Comment by GermainBergeron** [comment]
I want to point out that the page don't throw anymore but the tooltip is nowhere to be found

File: packages/react-vapor/src/components/drop/DropPod.tsx (hunk 1 above)
Comment (dmgauthier) [reply packages/react-vapor/src/components/drop/DropPod.tsx]: quite true

**Comment by dmgauthier** [comment]
> I want to point out that the page don't throw anymore but the tooltip is nowhere to be found

Good Catch, I will look into that.

**Pull request merged**